bodhi/server/models.py

```diff
@@ -43,8 +43,8 @@
 from bodhi.server.config import config
 from bodhi.server.exceptions import BodhiException, LockedUpdateException
 from bodhi.server.util import (
-    avatar as get_avatar, build_evr, flash_log, get_critpath_components,
-    get_nvr, get_rpm_header, header, tokenize, pagure_api_get)
+   avatar as get_avatar, build_evr, flash_log, get_critpath_components,
+    get_nvr, get_rpm_header, header, packagename_from_nvr, tokenize, pagure_api_get)
```

Comment:
```flake8``` is upset about the indentation of these two lines:

```
/home/vagrant/bodhi/bodhi/server/models.py:46:4: E121 continuation line under-indented for hanging indent
/home/vagrant/bodhi/bodhi/server/models.py:47:5: E131 continuation line unaligned for hanging indent
F.
======================================================================
FAIL: Enforce PEP-8 compliance on the codebase.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/vagrant/bodhi/bodhi/tests/test_style.py", line 37, in test_code_with_flake8
    self.assertEqual(subprocess.call(flake8_command), 0)
AssertionError: 1 != 0
```

I think line 46 had the correct indentation previously.